src/gui/layertree/qgslayertreeviewdefaultactions.cpp

```diff
@@ -342,6 +349,27 @@ void QgsLayerTreeViewDefaultActions::makeTopLevel()
   }
 }
 
+void QgsLayerTreeViewDefaultActions::moveToTop()
+{
+  QMap <QgsLayerTreeGroup *, int> groupInsertIdx;
+  int insertIdx;
+  Q_FOREACH ( QgsLayerTreeNode *n, mView->selectedNodes() )
```

Comment:
We are trying to not use `Q_FOREACH` in new code, would you please use a `for` range loop here?

Reply:
@MieWinstrup FYI: https://github.com/qgis/QGIS-Enhancement-Proposals/issues/103